api/passport/login.js

```diff
@@ -16,16 +16,17 @@ module.exports = function (passport) {
     },
         // Request is automatically passed to the passport 
         // We must call done which is like the response to the passport
-        function (req, email, password, done) {
+        function (req, username, password, done) {
```

Comment:
What was the reason for changing the param from `email` to `password`?

Reply:
![image](https://user-images.githubusercontent.com/29721784/38471700-bb9990f8-3b29-11e8-832e-4f0bad5491d5.png)
i saw this example in the passport documentation. I tried to match the example by assigning the different parameter to usernameField, but continue using username as the variable in the callback functions

Reply:
signup route should match